.gitlab/ci/.gitlab-ci.yml

```diff
@@ -1,6 +1,6 @@
 
 default:
-  image: docker-io.art.code.pan.run/devdemisto/gitlab-content-ci:1.0.0.26243
+  image: docker-io.artifactory.pan.local/devdemisto/gitlab-content-ci:1.0.0.27109
```

Comment:
any reason for that?

Reply:
yes, Im changing it to run with python 3.10. see https://github.com/demisto/dockerfiles/pull/7090/files